packages/reakit/src/Grid/index.ts

```diff
@@ -0,0 +1,14 @@
+import Grid from "./Grid";
+import GridItem from "./GridItem";
+
+export interface GridComponents {
+  Item: typeof GridItem;
+}
+
+const G = Grid as typeof Grid & GridComponents;
+
+G.Item = GridItem;
+
+export * from "./GridItem";
```

Comment:
We need both:
```js
export * from "./Grid";
export * from "./GridItem";
```

Otherwise, we can't do this:
```js
import { GridProps, GridItemProps } from "reakit";
```